src/globals.cpp

```diff
@@ -348,6 +348,10 @@ CTweak<bool> enableCanonicalTxOrder("consensus.enableCanonicalTxOrder",
 
 CTweak<unsigned int> numMsgHandlerThreads("net.msgHandlerThreads", "Max message handler threads", 0);
 CTweak<unsigned int> numTxAdmissionThreads("net.txAdmissionThreads", "Max transaction mempool admission threads", 0);
+CTweak<unsigned int> unconfPushAction("net.unconfChainResendAction",
+    "Action to take when this node thinks that a peer will now accept a previously unacceptable unconfirmed transaction"
+    "0: do not resend, 1: send an INV, 2: send the TX",
+    DEFAULT_COINBASE_RESERVE_SIZE);
```

Comment:
if memory serves this `const` is set to 1000, I guess this is not what you need here, unless the description of the tweet is outdated.

Reply:
no, its a copy-paste error. thanks for this catch!